src/app/utils/tests/MGPSet.spec.ts

```diff
@@ -78,4 +78,16 @@ describe('MGPSet', () => {
             expect(set.isEmpty()).toBeFalse();
         });
     });
+    describe('remove', () => {
```

Comment:
add un test "should throw if the Set was immutable !

Reply:
y'a pas encore d'immutable ici !